src/main/java/com/powsybl/ws/commons/computation/utils/SpecificationUtils.java

```diff
@@ -26,6 +27,7 @@
  * @author Kevin Le Saulnier <kevin.lesaulnier@rte-france.com>
  */
 public final class SpecificationUtils {
+    public static final int MAX_IN_CLAUSE_SIZE = 500;
```

Comment:
```suggestion
  /**
   * Maximum values per IN clause chunk to avoid StackOverflow exceptions.
   * Current value (500) is a safe default but can be changed
   */
    public static final int MAX_IN_CLAUSE_SIZE = 500;
```